docs/source/getting-started/installation.rst

```diff
@@ -16,8 +16,63 @@ You need to install:
 
    As part of the installation process you may need to set up the ``JAVA_HOME`` variable. You can find instructions for this process on :doc:`Windows,macOS, and Linux here <path-variables>`.
 
-2. Installing the SDK
-*********************
+2. Choosing Daml Enterprise or Daml Open Source
+***********************************************
+
+Daml comes in two variants: Daml Enterprise or Daml Open-Source. Both include the best in class SDK, Canton and all of the components that you need to write and deploy multi-party 
+applications in production, but they differ in terms of enterprise and non-functional capabilities:
+
+
+.. list-table::
+   :widths: 20 10 10
+   :header-rows: 1
+
+   * - Capability
+     - Enterprise
+     - Open Source
+   * - Daml Synchronisation
+     - Yes
+     - Yes
+   * - Sub-Transaction Privacy
+     - Yes
+     - Yes
+   * - `Transaction Processing <https://docs.daml.com/canton/architecture/overview.html#node-scaling>`_
+     - Parallel (fast)
+     - Sequential (slow)
+   * - `High Availability <https://docs.daml.com/canton/usermanual/ha.html>`_
+     - Yes
+     - No
+   * - High Throughput via Microservices
```

Comment:
Is there documentation for the "microservices" deployment?

Reply:
I'm changing this to 'Horizontal scalability' as requested on the slack conversation [here](https://daholdings.slack.com/archives/C973Z3TK2/p1648645300831909) and linking to [here](https://www.canton.io/docs/dev/user-manual/usermanual/ha.html#sequencer)